src/Cdn/Cdn.Autorest/examples/Invoke-AzCdnAbortProfileToAFDMigration.md

```diff
@@ -0,0 +1,11 @@
+### Example 1: Abort the Migration
+```powershell
+Invoke-AzCdnAbortProfileToAFDMigration -ProfileName cli-test-profile -ResourceGroupName cli-test-rg
+```
+
+```output
```

Comment:
If the output is empty dict, please ignore it. BTW, consider rephrasing to `Abort the profile migration to Azure Front Door` for clarity.